README.md

```diff
@@ -20,7 +23,12 @@ Add the plugin to the plugins list in your `.babelrc` file
 
 ```
 {
-  plugins: ['react-import-extends', ...otherPlugins]
+  plugins: [
+    ...otherPlugins,
+    ["react-import-extends", {
+      extends: "PureComponent" // not necessary, defaults to Component
```

Comment:
This can potentially break the app. Extending `PureComponent` in places where it should not have been extended. I was looking for a more intelligent way of doing it.